api/swagger.yaml

```diff
@@ -1746,7 +1746,7 @@ definitions:
       Created:
         description: |
           Date and time at which the image was created, formatted in
-          [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) format with nano-seconds.
+          [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) format with nano-seconds, or empty if the field was not set in the image config.
```

Comment:
We should also note this change in `version-history.md`

Reply:
Doh, good catch, I'll update (but I might not get to it today, so any other maintainer should feel free to take over :sweat_smile:).